master/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -47,184 +42,158 @@ container to network AB.
 
 ## Managing Calico policy for a network
 
-This section walks through an example of creating a docker network (with
+This section walks through two examples of creating a docker network (with
 Calico) and using the `calicoctl` command line interface to modify the policy
-associated with that network.
+associated with that network.  One example applies policy directly through the
+profile, the other applies policy through label selection.  Both approaches
+may be used together - in which case the selector based policy is applied first.
+
+### Global policy applied through label selection
 
-#### Create a Docker network
+Suppose we want to provide network policy for a set of database containers and a
+set of app containers, restricting the connectivity to allow only
+TCP connections from the app containers to the database containers connecting to 
+port 3306.
 
-To create a Docker network using Calico, run the `docker network create`
-command specifying `calico` as the network driver and `calico-ipam` as the IPAM driver.
+One way to do this would be to define two Docker networks, one for apps and the
+other for databases.  By modifying the profiles associated with each network, we
+can apply labels to endpoints in each network and then create some policy rules 
```

Comment:
This point needs to be emphasized more. The whole basis of this  approach is to leverage profiles just to apply labels.

I'm thinking it may make more sense to introduce Profile based policy first, point out its limitations, and then move on to how to do label-based policy instead